### PR TITLE
Devnet proper docker networking

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

**Review by cor — APPROVED** [review]
LGTM

**Review by cor — CHANGES_REQUESTED** [review]
`devnet-up` is still not workign:

` nix run "github:ComposableFi/composable/166638928088e9fb6204ce160951de8a02b35c47#devnet-up"`

> trace: warning: The check argument to evalModules is deprecated. Please set config._module.check instead.
error: getting status of '/nix/store/fykv528hph4vyb8nsfv1ijzb6ba20da4-source/.nix/services/subsquid-db.nix': No such file or directory
(use '--show-trace' to show detailed location information)

can you please fix this?

File: .nix/services/subsquid-indexer-gateway.nix [hunk 1]
```diff
@@ -1,8 +1,7 @@
-{ database }: {
+{ status, database, graphql-port }: {
```
Comment (andor0) [inline .nix/services/subsquid-indexer-gateway.nix:1]:
```suggestion
{ status-service-host, database, graphql-port }: {
```

File: scripts/polkadot-launch/network-builder.nix [hunk 2]
```diff
@@ -64,16 +64,17 @@ in rec {
...
     mk-chain {
       inherit bin;
       inherit chain;
       inherit port;
       inherit wsPort;
       inherit count;
       inherit nodeNames;
-      flags = [ "--rpc-cors=all" "--beefy" "--enable-offchain-indexing=true" ];
+      flags = [ "--rpc-cors=all" "--beefy" "--enable-offchain-indexing=true" ]
+        ++ flags;
```
Comment (andor0) [inline scripts/polkadot-launch/network-builder.nix:77]:
```suggestion
        ++ builtins.filter (flag: !builtins.elem flag ["--rpc-cors=all" "--beefy" "--enable-offchain-indexing=true"]) extraFlags
```

**Review by andor0 — APPROVED** [review]

File: scripts/polkadot-launch/network-builder.nix (hunk 2 above)
Comment (hussein-aitlahcen) [reply scripts/polkadot-launch/network-builder.nix]: Nice catch!

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Comment by hussein-aitlahcen** [comment]
@megifyio refresh

**Comment by hussein-aitlahcen** [comment]
@mergifyio update

**Pull request merged**